prow/cmd/gerrit/main.go

```diff
@@ -45,12 +45,13 @@ type options struct {
 	config            configflagutil.ConfigOptions
 	// lastSyncFallback is the path to sync the latest timestamp
 	// Can be /local/path, gs://path/to/object or s3://path/to/object.
-	lastSyncFallback       string
-	dryRun                 bool
-	kubernetes             prowflagutil.KubernetesOptions
-	storage                prowflagutil.StorageClientOptions
-	instrumentationOptions prowflagutil.InstrumentationOptions
-	changeWorkerPoolSize   int
+	lastSyncFallback        string
+	dryRun                  bool
+	kubernetes              prowflagutil.KubernetesOptions
+	storage                 prowflagutil.StorageClientOptions
+	instrumentationOptions  prowflagutil.InstrumentationOptions
+	changeWorkerPoolSize    int
+	maxChangeWorkerPollSize int
```

Comment:
I am not using tide in gerrit mode, so I do not know it's problems but I am curious about rationale behind that change. I see we have `changeWorkerPoolSize`, why do we need the maximum number then? Can't we just initiate a given number of workers?